<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="Book">
    <property name="title">
     <string>图书</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="deleteAction"/>
    <addaction name="saveAction"/>
    <addaction name="returnAction"/>
   </widget>
   <widget class="QMenu" name="Op">
    <property name="title">
     <string>审阅</string>
    </property>
    <addaction name="searchAction"/>
    <addaction name="sortAction"/>
    <addaction name="statsAction"/>
    <addaction name="exportAction"/>
    <addaction name="resetAction"/>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="infoAction"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="Book"/>
   <addaction name="Op"/>
   <addaction name="Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newAction">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="searchAction">
   <property name="text">
    <string>查询</string>
   </property>
  </action>
  <action name="sortAction">
   <property name="text">
    <string>排序</string>
   </property>
  </action>
  <action name="statsAction">
   <property name="text">
    <string>统计</string>
   </property>
  </action>
  <action name="exportAction">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="infoAction">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="returnAction">
   <property name="text">
    <string>返回主界面</string>
   </property>
  </action>
  <action name="resetAction">
   <property name="text">
    <string>查看全部</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
